macros/CSS_Ref.ejs

```diff
@@ -159,10 +161,12 @@ if (types.some(function(type) { return type === "syntaxes"; })) {
             });
         }
 
-        // Add items within at-rule syntax
-        var items = getItemsFromSyntax(data.syntaxes[syntax]);
+        // Add items within at-rule syntax (e.g. "@annotation")
+        // check whether https://github.com/mdn/data/pull/66 was merged
+        var items = getItemsFromSyntax(syntaxes[syntax].syntax || syntaxes[syntax]);
```

Comment:
👍 works with current data, and appears that it will work with https://github.com/mdn/data/pull/66